src/Utilities/DynamicFieldsUtils.res

```diff
@@ -1072,12 +1073,23 @@ let getPaypalRequiredFields = (
       }
     | ShippingAddressCountry(_) => details.shippingAddress.countryCode->Option.getOr("")
     | ShippingAddressPincode => details.shippingAddress.postalCode->Option.getOr("")
-    | Email => details.email
+    | Email =>
+      switch item.required_field {
+      | "payment_method_data.billing.email"
```

Comment:
Is there some other way instead of hard coding the path. Since later on if this path gets changed from the backend, this code will start breaking?